SmartDeviceLink/SDLRadioControlCapabilities.m

```diff
@@ -15,6 +15,26 @@ - (instancetype)initWithModuleName:(NSString *)moduleName radioEnableAvailable:(
     if(!self){
         return nil;
     }
+
+    self.moduleName = moduleName;
```

Comment:
The deprecated initializer should just call the new initializer with the new options set to `@NO`